core/dbt/include/global_project/macros/adapters/common.sql

```diff
@@ -130,6 +130,29 @@
   {{ return(adapter_macro('alter_column_type', relation, column_name, new_column_type)) }}
 {% endmacro %}
 
+
+
+{% macro alter_column_comment(relation, column_dict) -%}
+  {{ return(adapter_macro('alter_column_comment', relation, column_dict)) }}
+{% endmacro %}
+
+{% macro default__alter_column_comment(relation, column_name, new_column_type) -%}
```

Comment:
I think this should be `relation, column_dict`, not `relation, column_name, new_column_type`!

Reply:
Good catch, this is fixed now